### PR TITLE
docs(windows): add knowledge base article for distributed keyboard installation

### PR DESCRIPTION
**Pull request by rc-swag**
The steps to install a keyboard and then enable it for non-admin users or even each user on the local machine is not documented. This knowldege base article addresses that.

### CODE REVIEW

File: knowledge-base/kb0112.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     - Replace 'BCP47Tag' with the language tag, which can be found in Keyman Configuration.
+
+3. **Example:**
+   - As an example, let's consider the installation of the BJ-OjiCree keyboard (Kingfisher):
+     - KeyboardID: `bj_cree_east`
+     - BCP47Tag: `crl-Cans-CA`
+   - Installation commands:
+     - Elevated session:
+       ```
+       kmshell -s -i bj_cree_east.kmp.kmp
```
Comment (darcywong00) [inline knowledge-base/kb0112.md:39]: which `.kmp`s should be here?

Comment (rc-swag) [reply knowledge-base/kb0112.md:39]: thanks good pick up I was mergeing to examples.

Comment (rc-swag) [reply knowledge-base/kb0112.md:43]: I also added a description on how to identify the keyboard id and language name. I aslo fixed the last sentence of this  md

Comment (darcywong00) [reply knowledge-base/kb0112.md:43]: gotcha. so should this be bj_cree_east.kmp?

File: knowledge-base/kb0112.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Keyman for Windows: Distributed Installation of Keyboards
+
+## Installing Keyman Package Files using kmshell.exe
+
+**Introduction:**
+This knowledge base provide the steps to to install keyboards remotely and enable them for a non-elevated user. The key is that the registration for language and the keyboard needs to be run as the user who needs the keyboard enabled.
```
Comment (mcdurdin) [inline knowledge-base/kb0112.md:6]:
```suggestion
This article documents the process for installation of a Keyman keyboard package via command line or script. This is particularly helpful when deploying keyboards to multiple computers. The process is split into an elevated step to install the keyboard and register it in the system, and a current-user step, to link the keyboard to a language for the user.
```

**Review by darcywong00 — APPROVED** [review]
lgtm

File: knowledge-base/kb0112.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   - Ensure the previous kmshell process has completed before running the next command.
+   - Run the following command in a non-elevated command prompt for each user:
+     ```
+     kmshell -s -ikl KeyboardID BCP47Tag
+     ```
+     - Replace 'KeyboardID' with the appropriate identifier for the keyboard (usually the same as the package filename without `.kmp`).
+     - Replace 'BCP47Tag' with the language tag, which can be found in Keyman Configuration.
+
+3. **Example:**
+   - As an example, let's consider the installation of the **BJCreeUNI** (east) keyboard.
```
Comment (mcdurdin) [inline knowledge-base/kb0112.md:33]:
```suggestion
   - As an example, let's consider the installation of the **BJCreeUNI (east)** keyboard.
```

File: knowledge-base/kb0112.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+     Normally, the KeyboardID will be the same as the package filename (sans .kmp). You can check it by installing the package locally and viewing the details in Keyman Configuration. Similarly, the BCP 47 tag (e.g.en-US) can be found in Keyman Configuration installed keyboard’s details looking at the languages label.
```
Comment (mcdurdin) [inline knowledge-base/kb0112.md:34]:
```suggestion
     Normally, the KeyboardID will be the same as the package filename (sans .kmp). You can check it by installing the package locally and viewing the details in Keyman Configuration. Similarly, the BCP 47 tag (e.g.en-US) can be found in Keyman Configuration installed keyboard’s details, looking at the languages label.
```

File: knowledge-base/kb0112.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# Keyman for Windows: Distributed Installation of Keyboards
```
Comment (mcdurdin) [inline knowledge-base/kb0112.md:1]:
```suggestion
# Keyman for Windows: Scripted Installation of Keyboards
```

**Review by mcdurdin — APPROVED** [review]
Suggest we call it 'scripted installation' rather than 'distributed' because we are really only still talking about the process for a single machine. The distributed bit is out of scope.

I've made some editorial suggestions, feel free to pick and choose.

LGTM

**Review by ermshiperete — APPROVED** [review]

**Pull request merged**